hw10/fyn_calc.py

```diff
@@ -0,0 +1,47 @@
+def calc(number1, number2, operator):
+    result = None
+    if operator == '+':
+        result = number1 + number2
+    elif operator == '-':
+        result = number1 - number2
+    elif operator == '*':
+        result = number1 * number2
+    elif operator == '/':
+        if number2 == 0:
+            print("Stop! You can't divide by zero")
+            return
+        result = number1 / number2
+    else:
+        print("Incorrect operation")
+    return result
+
+
+def operations():
+    operator = input("Enter operator(Enter '+', '-', '*', '/'): ")
```

Comment:
input() better use outside of functions. You function operations() should receive parameter operation: str